src/message/header/mod.rs

```diff
@@ -131,22 +137,47 @@ impl Display for Headers {
     }
 }
 
+/// A possible error when converting a `HeaderName` from another type.
+// comes from `http` crate
+#[allow(missing_copy_implementations)]
+pub struct InvalidHeaderName {
```

Comment:
I agree with skipping the `Copy` implementation, but `Clone` should be fine here.

Reply:
indeed :+1: